add realization of reverse string

AnnLvu opened the pull request.

AnnLvu committed:
add realization of reverse string

KITKATJN commented on src/main/java/main/StringReverse.java:
```diff
@@ -1,15 +1,47 @@
 package main;
 
+import java.sql.SQLOutput;
```
Do you really need this import?
- [ ] remove all unused imports

AnnLvu replied:
removed

KITKATJN commented on src/main/java/main/StringReverse.java:
```diff
@@ -1,15 +1,47 @@
+    public static void reverseArray(char[] charArray) {
+        {
+            for (int i = charArray.length - 1; i >= 0; i--) {
+                for (int j = 0; j < i; j++) {
```
- [ ] Please rewrite this method, using only one cycle "for"

AnnLvu replied:
rewrite

KITKATJN commented on src/main/java/main/StringReverse.java:
```diff
@@ -1,15 +1,47 @@
+        }
+
+        char[] charArray = stringToReverse.toCharArray();
+        reverseArray(charArray);
```
you can make "reverseArray" returning String

AnnLvu replied:
made "reverseArray" returning String

KITKATJN commented on src/main/java/main/StringReverse.java:
```diff
@@ -1,15 +1,47 @@
+    }
+
+
+    public static void reverseArray(char[] charArray) {
```
- [ ] change void to String

AnnLvu replied:
change

AnnLvu committed:
Merge pull request #1 from KITKATJN/master

Update test.yml

AnnLvu committed:
removed unused import

AnnLvu committed:
Merge remote-tracking branch 'origin/master'

AnnLvu committed:
removed unused import

AnnLvu committed:
removed unused import
change void to String
rewrite method reverseArray, using only one cycle "for"
make "reverseArray" returning String
